core-web/libs/sdk/react/src/lib/hooks/useDotcmsEditor.ts

```diff
@@ -49,7 +51,7 @@ export const useDotcmsEditor = ({ pageContext, config }: DotcmsPageProps) => {
      * Reloads the page when changes are made in the editor.
      */
     useEffect(() => {
-        const insideEditor = isInsideEditorFn();
+        const insideEditor = !!getUVEState();
```

Comment:
this means that in all the modes in UVE we need this to be `true`, correct?

Reply:
Correct, the presence of a response means we are inside the `UVE` but I can add a property for that as I commented above

Reply:
Even tho the uve state is a good idea, looks like we still need some kind of function to know when we're editing...

Reply:
Yes, we can get discuss that when we start to make this changes, I rollbacked this